src/ServiceBuilder.php

```diff
@@ -92,7 +92,7 @@ public function __construct(array $config = [])
      * ```
      * use Google\Cloud\ServiceBuilder;
      *
-     * // Create a storage client using application default credentials.
+     * // Create a Storage client using application default credentials.
```

Comment:
Google CloudPlatform Client seems verbose, given the context. The comment to me is descriptive of both lines together.